automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/README.md

```diff
@@ -0,0 +1,173 @@
+# AWS Health AWS_RISK_CREDENTIALS_EXPOSED
+
+This example demonstrates how to use [AWS Step Functions](https://aws.amazon.com/step-functions/) to orchestrate a serverless [AWS Lambda](http://aws.amazon.com/lambda/) workflow in response to an [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) Event generated by [AWS Health](http://docs.aws.amazon.com/health/latest/ug/what-is-aws-health.html). 
+
+AWS proactively monitors popular code repository sites for exposed IAM access keys. On detection of an exposed IAM access key, AWS Health generates an AWS_RISK_CREDENTIALS_EXPOSED CloudWatch Event. In response to this event, an automated workflow deletes the exposed IAM Access Key, summarizes the recent API activity for the exposed key, and sends the summary message to an SNS Topic to notify the subscribers- all orchestrated by an AWS Step Functions state machine.
+
+This repository contains sample code for all the Lambda functions depicted in the diagram below as well as an AWS CloudFormation template for creating the functions and related resources.
+
+![screenshot for instruction](images/Architecture.png)
+
+### Walkthrough of the Architecture
+1. An IAM Access Key is inadvertently uploaded to one of the popular code repositories monitored by AWS.
+1. AWS Health detects the key, generating a AWS_RISK_CREDENTIALS_EXPOSED CloudWatch Event. AWS also temporarily restricts the API calls the key is able to make.
+1. A configured CloudWatch Events rule matches this event, triggering an execution of the `ExposedKey` state machine, which has the following sub-steps:
+    * Delete the exposed IAM Access Key Pair
+    * Summarize recent API activity for the user from CloudTrail
+    * Send security notification message containing summary to `SecurityNotificationTopic` SNS Topic.
+
+## Running the Example
+ 
+#### Option 1: Launch the CloudFormation Template in US East - N. Virginia (us-east-1) 
+The backend infrastructure can be deployed in US East - N. Virginia (us-east-1) using the provided CloudFormation template.
+Click **Launch Stack** to launch the template in the US East - N. Virginia (us-east-1) region in your account:
+
+[![Launch Stack into N. Virginia with CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=AWSHealthCredsExposed&templateURL=https://s3.amazonaws.com/aws-health-tools-projects/AWS-RISK-CREDENTIALS-EXPOSED/risk_credentials_exposed.output.yaml)
```

Comment:
When deploying the CFN template I get this error when it tries to create the lambda functions: "Error occurred while GetObject. S3 Error Code: AuthorizationHeaderMalformed. S3 Error Message: The authorization header is malformed; the region 'us-east-1' is wrong; expecting 'us-west-2'". Seems like it cannot find the code? I put some more comments in the template as well